libraries/ZigBeeNet/Database/ZigBeeNetworkDatabaseManager.cs

```diff
@@ -0,0 +1,292 @@
+using Serilog;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
```

Comment:
Is it really necessary?

Reply:
I mean the using System.Threading , beause if you use a Task it should be enough to import System.Threading. Task. But I am not sure

Reply:
done